### PR TITLE
CHANGELOG: Document changes since v0.2.0

### PR DESCRIPTION
**Pull request by wking**
I've also updated the v0.2.0 docs to soften the claims about staged installs, since v0.3.0 adds "loading assets from user-edited files" while v0.2.0 only had "remembers previously-generated assets".

CC @abhinavdahiya, @crawford

### CODE REVIEW

**Comment by wking** [comment]
[images][1]:

```
2018/10/20 04:54:24 Copied 0.88Mi of artifacts from release-latest to /logs/artifacts/release-latest
rpc error: code = 2 desc = oci runtime error: exec failed: container_linux.go:247: starting container process caused "process_linux.go:110: decoding init error from pipe caused \"read parent: connection reset by peer\""
```

[e2e-aws][2]:

```
18/10/20 04:54:30 Ran for 2m47s
error: could not run steps: failed to wait for release pod to complete: could not wait for pod completion: pod release-latest was already deleted
```

/retest

[1]: https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_installer/504/pull-ci-openshift-installer-master-images/615/build-log.txt
[2]: https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_installer/504/pull-ci-openshift-installer-master-e2e-aws/846/build-log.txt

**Comment by wking** [comment]
Rebased onto master and updated to set the release date to Monday with 7c09662 -> 40fc60e.

**Comment by wking** [comment]
Rebased onto master and updated to set the release date to Monday with 7c09662 -> 40fc60e.

**Comment by wking** [comment]
[Hung cluster][1]:

```
2018/10/22 04:21:57 Container setup in pod e2e-aws completed successfully
2018/10/22 06:21:03 Container teardown in pod e2e-aws completed successfully
2018/10/22 06:21:14 Copying artifacts from e2e-aws into /logs/artifacts/e2e-aws
Another process exited
```

/retest

[1]: https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_installer/504/pull-ci-openshift-installer-master-e2e-aws/803/build-log.txt

**Comment by wking** [comment]
I've pushed 40fc60e -> b09a86c documenting the new `delete` command and its subcommands and the deprecation of `delete-cluster` (all from #493).  Those seem like important, user-facing changes ;).

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,42 @@ All notable changes to this project will be documented in this file.
...
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## 0.3.0 - 2018-10-22
+
+### Added
+
+- Asset state is loaded from the install directory, allowing for a [staged
+  install](docs/user/overview.md#multiple-invocations).
+- The installer pushes a `bootstrap-complete` event to the
+  `kube-system` namespace when the boostrapping process completes and
```
Comment (crawford) [inline CHANGELOG.md]: There is a bug with this. The bootstrap reports completed long before it actually completes :(

**Pull request merged**